### PR TITLE
Chore/coverage

### PR DESCRIPTION
**Pull request by SimoneLazzaris**
I've added some test function to cover corner cases

### CODE REVIEW

File: cmd/helper/error.go [hunk 1]
```diff
@@ -25,10 +25,12 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+var osexit = os.Exit
```
Comment (byo) [inline cmd/helper/error.go:28]: This looks like a quite intrusive change since - IMO we should first try to increase the code coverage without altering the immudb code and only if that won't bring us above certain % threshold then use more intrusive methods.

Can you please extract tests that require this `osexit` into a separate PR? That way we'll still be able to merge the majority of this PR.

Comment (jeroiraz) [reply cmd/helper/error.go:28]: However, it's on the command line components, if there is no easy way for dealing with it, I think it'd be ok

Comment (SimoneLazzaris) [reply cmd/helper/error.go:28]: Unfortunately, there is a relatively big chunk of code that fails with "os.Exit", which makes it impossible to account for with coverage. Hence my modification, which is really very localized, but I can see your point.

Of course I can extract those tests into a different PR.

Comment (byo) [reply cmd/helper/error.go:28]: Ok, as Jeronimo said, I guess it's acceptable - there's likely no other way to test this code.

**Review by byo — APPROVED** [review]

**Review by byo — APPROVED** [review]
Good job! We're green on coverage now!
